src/app/application/components/route-view/RouteTitle.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <template
+    v-if="false"
+  >
+    {{ props.title }}
+  </template>
+</template>
+<script lang="ts" setup>
+import { inject, PropType, watch, onUnmounted } from 'vue'
+
+import { RouteView } from './RouteView.vue'
+
+const props = defineProps({
+  title: {
+    type: String as PropType<string>,
+    required: false,
+    default: '',
+  },
+})
+
+const symbol = Symbol('route-title')
+const routeView: RouteView | undefined = inject('route-view-parent')
+if (typeof routeView !== 'undefined') {
+  watch(() => props.title, (title: string) => {
```

Comment:
**Suggestion**: Watcher callbacks always infer the type of its parameters based on what’s watched so they shouldn’t ever need to be annotated.

```suggestion
  watch(() => props.title, (title) => {
```